besspin/cyberPhys/ui/hacker-kiosk/zmq_python_test.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import zmq
+import time
+import json
+import random
+
+context = zmq.Context()
+socket = context.socket(zmq.PAIR)
+socket.bind("tcp://*:3333")
```

Comment:
I think we can remove this file and use https://github.com/GaloisInc/BESSPIN-Tool-Suite/blob/develop/besspin/cyberPhys/hackerKiosk/hackerKiosk.py as your backend.